src/main/java/org/teamresistance/frc/IO.java

```diff
@@ -17,6 +18,9 @@
 
   public static final Gyro3D gyro = new Gyro3D(SPI.Port.kMXP);
 
+  public static final Solenoid grabSolenoid = Hardware.Solenoids.doubleSolenoid(1, 2, Solenoid.Direction.EXTENDING);
+  public static final Solenoid armSolenoid = Hardware.Solenoids.doubleSolenoid(3, 4, Solenoid.Direction.EXTENDING);
```

Comment:
Weird how they don't support a traditional Solenoid. I checked the issue tracker and no one seems to have noticed, awk.

Do you think we should go ahead and implement our our single-solenoid wrapper for the WPILib Solenoid and keep all the rest of the code the same?

```java
// SingleSolenoid.java:
public class SingleSolenoid implements Solenoid {
   
  public SingleSolenoid(int channel) {
    // Instantiate a WPILib Solenoid
  }
}

// IO.java:
public static final Solenoid grabSolenoid = new SingleSolenoid(1);
```